### PR TITLE
Add endpoint for online users

### PR DESCRIPTION
**Pull request by MariaRoz**

### CODE REVIEW

File: src/messages/messages.controller.ts [hunk 1]
```diff
@@ -2,30 +2,34 @@ import { Body, Controller, Delete, Get, Param, Post, Req, UseGuards } from '@nes
...
     return this.service.getMessageById(id);
   }
 
-  @Delete(':id')
+  @Delete('messages/:id')
   deleteMessage(@Param('id') id: number) {
     return this.service.delete(id);
   }
+
+  @Get('online')
```
Comment (myownsumm) [inline src/messages/messages.controller.ts]: That endpoint shouldn't refer to /messages namespace

File: src/messages/messages.service.ts [hunk 2]
```diff
@@ -36,4 +36,11 @@ export class MessagesService {
   async delete(id: number): Promise<DeleteResult> {
      return this.messageRepository.delete(id);
   }
+
+  async getOnlineUsers(): Promise<string []> {
```
Comment (myownsumm) [inline src/messages/messages.service.ts]: MessagesService & Users online? That is wrong place to keep it.

File: src/messages/messages.service.ts [hunk 3]
```diff
@@ -36,4 +36,11 @@ export class MessagesService {
...
+    const result = await this.messageRepository.find({relations: ['author']});
+
+    return [...new Set(result.filter(messages => (new Date().getTime() - messages.createdAt.getTime()) / 60000 < 15)
```
Comment (myownsumm) [inline src/messages/messages.service.ts]: Use ORM to build correct query, not JS.

Comment (MariaRoz) [reply src/messages/messages.service.ts]: Fixed (Using QueryBuilder)

File: src/users/users.module.ts [hunk 4]
```diff
@@ -2,10 +2,13 @@ import { Module } from '@nestjs/common';
 import { UsersService } from './users.service';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { UserEntity } from './user.entity';
+import { UserController } from './user.controller';
+import { Message } from '../messages/messages.entity';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([UserEntity])],
+  imports: [TypeOrmModule.forFeature([UserEntity, Message])],
```
Comment (myownsumm) [inline src/users/users.module.ts]: MessageEntity

Comment (MariaRoz) [reply src/users/users.module.ts]: Fixed

**Pull request merged**